ovos_PHAL_plugin_commonIOT/device_manager.py

```diff
@@ -35,6 +35,79 @@ def __init__(self, bus):
         # self.bus.on("ovos.iot.device.mute", self.handle_mute)
         # self.bus.on("ovos.iot.device.unmute", self.handle_unmute)
 
+        # iot lights actions
+        self.bus.on("ovos.iot.device.get.brightness", self.handle_get_brightness)
+        self.bus.on("ovos.iot.device.set.brightness", self.handle_set_brightness)
+        self.bus.on("ovos.iot.device.decrease.brightness", self.handle_decrease_brightness)
+        self.bus.on("ovos.iot.device.increase.brightness", self.handle_increase_brightness)
+        self.bus.on("ovos.iot.device.get.color", self.handle_get_color)
+        self.bus.on("ovos.iot.device.set.color", self.handle_set_color)
+
+    # light handlers
+    def handle_get_brightness(self, message):
+        device = self.search_for_device(message.data.get("device"))
+        if device:
+            return device.brightness
+        return None
+
+    def handle_set_brightness(self, message):
+        device = self.search_for_device(message.data.get("device"))
+        if device:
+            try:
+                device.set_brightness(message.data.get("brightness"))
+                return device.brightness
+            except Exception as e:
+                LOG.error(e)
+                return e
+        return False
+
+    def handle_decrease_brightness(self, message):
+        device = self.search_for_device(message.data.get("device"))
+        if device:
+            try:
+                amount = int(message.data.get("amount"))
+                device.set_brightness(device.brightness - amount)
+                return device.brightness
+            except Exception as e:
+                LOG.error(e)
+                return e
+        return False
+
+    def handle_increase_brightness(self, message):
+        device = self.search_for_device(message.data.get("device"))
+        if device:
+            try:
+                amount = int(message.data.get("amount"))
+                device.set_brightness(device.brightness + amount)
+                return device.brightness
+            except Exception as e:
+                LOG.error(e)
+                return e
+        return False
+
+    def handle_get_color(self, message):
+        device = self.search_for_device(message.data.get("device"))
+        if device:
+            return device.color
+        return None
+
+    def handle_set_color(self, message):
+        device = self.search_for_device(message.data.get("device"))
+        if device:
+            try:
+                device.change_color(message.data.get("color"))
+                return device.color
+            except Exception as e:
+                LOG.error(e)
+                return e
+        return None
+
+    def search_for_device(self, device):
+        for d_id, d in self devices.items():
+            if device == d_id or device == d.name:
+                return d
```

Comment:
this works for now :+1:  later we will want to support fuzzy matching and allow devices to have aliases, but out of scope for this PR